docs/lessons/buttons.rst

```diff
@@ -0,0 +1,188 @@
+Buttons
+-------
+
+So far we have created code that makes the device do something. This is called
+*output*. However, we also need the device to react to things. Such things are
+called *input*.
+
+It's easy to remember: output is what the device puts out to the world
+whereas input is what goes into the device for it to process.
+
+The most obvious means of input on the micro:bit are its two buttons, labelled
+``A`` and ``B``. Somehow, we need MicroPython to react to button presses.
+
+This is remarkably simple::
+
+    from microbit import *
+
+    sleep(10000)
+    display.scroll(str(button_a.get_presses()))
+
+All this script does is sleep for ten thousand milliseconds (i.e. 10 seconds)
+and then scrolls the number of times you pressed button ``A``. That's it!
+
+While it's a pretty useless script, it introduces a couple of interesting new
+ideas:
+
+#. The ``sleep`` *function* will make the micro:bit sleep for a certain number
+   of milliseconds. If you want a pause in your program, this is how to do it.
+   A *function* is just like a *method*, but it isn't attached by a dot to an
```

Comment:
Yes, I just noticed I was reading it in the wrong order. Started from the beginning now :)